resources/views/auth/login.blade.php

```diff
@@ -22,48 +22,57 @@
 
 @section('auth_body')
     <form action="{{ $login_url }}" method="post">
-        {{ csrf_field() }}
+        @csrf
 
         {{-- Email field --}}
         <div class="input-group mb-3">
-            <input type="email" name="email" class="form-control {{ $errors->has('email') ? 'is-invalid' : '' }}"
-                   value="{{ old('email') }}" placeholder="{{ __('adminlte::adminlte.email') }}" autofocus>
+            <input type="email" name="email" class="form-control @error('email') is-invalid @enderror"
+                   value="{{ old('email') }}" placeholder="{{ __('adminlte::adminlte.email') }}"
+                   autocomplete="email" autofocus>
+
             <div class="input-group-append">
                 <div class="input-group-text">
                     <span class="fas fa-envelope {{ config('adminlte.classes_auth_icon', '') }}"></span>
                 </div>
             </div>
-            @if($errors->has('email'))
-                <div class="invalid-feedback">
-                    <strong>{{ $errors->first('email') }}</strong>
-                </div>
-            @endif
+
+            @error('email')
+                <span class="invalid-feedback" role="alert">
+                    <strong>{{ $message }}</strong>
+                </span>
+            @enderror
         </div>
 
         {{-- Password field --}}
         <div class="input-group mb-3">
-            <input type="password" name="password" class="form-control {{ $errors->has('password') ? 'is-invalid' : '' }}"
-                   placeholder="{{ __('adminlte::adminlte.password') }}">
+            <input type="password" name="password" class="form-control @error('password') is-invalid @enderror"
+                   placeholder="{{ __('adminlte::adminlte.password') }}" autocomplete="current-password">
```

Comment:
For security reasons, it will be better to disable autocomplete in login forms.

Reply:
OK, I wasn't sure about adding this. I added because it is used on the [laravel/ui](https://github.com/laravel/ui/blob/3.x/src/Auth/bootstrap-stubs/auth/login.stub) auth views (they also use it on some inputs of the breeze package). I will remove all autocomplete attributes, just in case...

Reply:
done!